src/Game.css

```diff
@@ -0,0 +1,107 @@
+body {
```

Comment:
This can go in the App.css as it is project wide

Reply:
This is resolved